docs/developer/decision-records/2022-07-01-service-authentication/README.md

```diff
@@ -0,0 +1,32 @@
+# Service authentication
+
+## Decision
+
+Access to the Registration Service API is authenticated.
+
+Callers must provide a Bearer token containing a JSON Web Token ([JWT](https://datatracker.ietf.org/doc/html/rfc7519)) transmitted as a JSON Web Signature ([JWS](https://www.rfc-editor.org/rfc/rfc7515)) signed with an ECDSA P-256 Elliptic Curve key.
+
+Tokens contain the following claims:
+
+| Claim           | Value                                                        |
+| --------------- | ------------------------------------------------------------ |
+| Issuer          | The participant [did:web](https://w3c-ccg.github.io/did-method-web/) identifier (example: `did:web:example.com`). The Issuer Claim must resolve to a DID Document containing the public key of the key pair used to sign the JWS. This allows the server to verify the token signature against the source's public key. |
+| Subject         | The fixed string `verifiable-credential`.                    |
+| Audience        | The Registration Service API base endpoint (example: `http://dataspace.example.com:8182/authority`). This allows the server to verify the intended audience. |
```

Comment:
I have a few questions:
What is an audience? How many different audience would we have inside one dataspace?
Why is this called audience if this is supposed to be a registration-service endpoint?

Why a caller would need to specify a registration service endpoint in a token when calling the registration service itself already?

Reply:
- audience is a claim as `aud` to identify the recipient for which a JWT is intended for. https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3 So I this case when a participant will interact with registration service then registration service is the audience.
- This claim is used here to make sure that JWT is intended for registration service which is verifying the signature and not for someone else.

Reply:
I see thanks!